microprofile.jdt/org.eclipse.lsp4mp.jdt.test/projects/maven/hibernate-orm-resteasy-yaml/src/main/java/org/acme/hibernate/orm/FruitResource.java

```diff
@@ -75,9 +75,13 @@ public Fruit update(@PathParam Integer id, Fruit fruit) {
     }
 
     @DELETE
-    @Path("{id}")
+    @Path(
```

Comment:
Maybe for this test it makes sense to switch the order of `@Path` and `@Transactional`, so that the multiline annotation is the one right before the method.